django_project/geosight/data/admin/base.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+"""
+GeoSight is UNICEF's geospatial web-based business intelligence platform.
+
+Contact : geosight-no-reply@unicef.org
+
+.. note:: This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+"""
+__author__ = 'zakki@kartoza.com'
+__date__ = '14/01/2024'
+__copyright__ = ('Copyright 2023, Unicef')
+
+
+class BaseAdminMixin:
```

Comment:
Please change the name to BaseAdminResourceMixin
To flag this admin is for Resource that contains creator, modified, etc

Reply:
Also please add extension with admin.ModelAdmin

Reply:
Also please add list_display(created_by, created_at, modified_by, modify_at) and can be concatenated on the child list_display